tilequeue/query.py

```diff
@@ -157,16 +157,17 @@ def __init__(self, conn_info, layer_data, io_pool, n_conn):
 
         self.dbnames = self.conn_info.pop('dbnames')
         self.dbnames_query_index = 0
-        self.sql_conn_pool = DBAffinityConnectionsNoLimit(
-            self.dbnames, n_conn, self.conn_info)
+        self.sql_conn_pool = DatabaseCycleConnectionPool(
+            n_conn, 100, self.dbnames, self.conn_info)
```

Comment:
How many active connections do we maintain per DB now?

How many are typically from a single tileserver or tilequeue instance / worker?